tests/test_material_results.py

```
@@ -85,13 +58,16 @@ def test_flows(solved_connection):
     Test that the emissions from each technology archetype are correct, and check total emissions
     """
     con, name, tech, period, flow_target = solved_connection
-    flow = (
+    row = (
         con.cursor()
         .execute(
             f"SELECT SUM(flow) FROM main.OutputFlowOut WHERE tech == '{tech}' AND period == {period}"
```

Ruff (0.14.3)

Possible SQL injection vector through string-based query construction

(S608)